velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2858,52 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+TEST_F(DateTimeFunctionsTest, currentTimestampTest) {
+  const auto evaluateCurrentTimestamp = [&](int64_t expectedTimestamp,
+                                            const std::string& timeZone) {
+    VectorPtr expected =
```

Comment:
This vector has only one element. The logic below needn't be written as a loop. It might be better to just retrieve the element at the 0th index of the vector and compare them. Same for the next test function also.